`azurerm_shared_image_version` - Support `replicated_region_deletion_enabled` and `target_region.exclude_from_latest_enabled` (#23147)
hashicorp/terraform-provider-azurerm: website/docs/r/shared_image_version.html.markdown

-> **NOTE:** You must specify exact one of `blob_uri`, `managed_image_id` and `os_disk_snapshot_id`.

> * `replicated_region_deletion_enabled` - (Optional)  Whether removing this Gallery Image Version from replicated regions is allowed. Defaults to `false`. Changing this forces a new resource to be created.

Reviewer: Presumably this is?

```suggestion
* `deletion_of_replicated_locations_enabled` - (Optional) Specifies whether this Shared Image Version can be deleted from the Azure Regions this is replicated to. Defaults to `false`. Changing this forces a new resource to be created.
```

Author: doc updated. Thanks.

* `disk_encryption_set_id` - (Optional) The ID of the Disk Encryption Set to encrypt the Image Version in the target region. Changing this forces a new resource to be created.

> * `exclude_from_latest_enabled` - (Optional) Whether excluding this Image Version from the `latest` filter. If set to `true` this Image Version won't be returned for the `latest` version. Defaults to `false`.

Reviewer: ```suggestion
* `exclude_from_latest_enabled` - (Optional) Specifies whether this Shared Image Version should be excluded when querying for the `latest` version. Defaults to `false`.
```

Author: doc updated. Thanks.
